### PR TITLE
style: Rewording, clarification, consistency of README 

### PR DESCRIPTION
**Pull request by mgrhm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+
+## Contact
+
+## Team
+
+Our team is made of three core members:
+
+- [Regis Freyd (djaiss)](https://github.com/djaiss)
+
+- [Théo Matthieu (mokto)](https://github.com/mokto)
```
Comment (asbiin) [inline README.md:286]: Théo is actually no longer part of the core members, we could remove this line.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+
+We are also fortunate to have an amazing [community of developers](https://github.com/monicahq/monica/graphs/contributors) who help us greatly.
+
+## Thank you, open source
+
+Monica uses a lot of open source projects and we thank them with all our hearts. We hope that providing Monica as an free, open source project will help other people the same way those softwares have helped us.
+
+## License
+
+Copyright (c) 2016–2020
```
Comment (asbiin) [inline README.md:298]: We should probably increase the year (yeah)
```suggestion
Copyright (c) 2016–2021
```

**Review by asbiin — APPROVED** [review]
That is an incredible work! Thank you @mgrhm !!

File: README.md (hunk 1 above)
Comment (mgrhm) [reply README.md]: Thanks for letting me know; I've made this change now.

File: README.md (hunk 2 above)
Comment (mgrhm) [reply README.md]: Good spot. Fixed.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,298 @@
+<p align="center"><img src="https://user-images.githubusercontent.com/61099/37693034-5783b3d6-2c93-11e8-80ea-bd78438dcd51.png"></p>
+<h1 align="center">Personal Relationship Manager</h1>
+
+<p align="center">
+<a href="https://dev.azure.com/monicahq/monica"><img src="https://dev.azure.com/monicahq/monica/_apis/build/status/monicahq.monica?branchName=master" alt="Build Status"></a>
+<a href="https://styleci.io/repos/82238168"><img src="https://styleci.io/repos/82238168/shield?branch=master" alt="StyleCI"></a>
+<a href="https://greenkeeper.io/"><img src="https://badges.greenkeeper.io/monicahq/monica.svg" /></a>
```
Comment (asbiin) [inline README.md:7]: Let's remove greenkeeper, it's not connected anymore.
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,297 @@
...
+<h1 align="center">Personal Relationship Manager</h1>
+
+<p align="center">
+<a href="https://dev.azure.com/monicahq/monica"><img src="https://dev.azure.com/monicahq/monica/_apis/build/status/monicahq.monica?branchName=master" alt="Build Status"></a>
+<a href="https://styleci.io/repos/82238168"><img src="https://styleci.io/repos/82238168/shield?branch=master" alt="StyleCI"></a>
+<a href="https://sonarcloud.io/project/activity?custom_metrics=coverage&amp;graph=custom&amp;id=monica"><img src="https://sonarcloud.io/api/project_badges/measure?project=monica&amp;metric=coverage" alt="Code coverage"/></a>
+<a href="https://sonarcloud.io/dashboard?id=monica"><img src="https://sonarcloud.io/api/project_badges/measure?project=monica&amp;metric=alert_status" alt="Quality gate" /></a>
+<a href="https://github.com/monicahq/monica/blob/master/LICENSE"><img src="https://img.shields.io/badge/License-AGPL-blue.svg" alt="License"></a>
+<a class="badge-align" href="https://slack.monicahq.com"><img src="https://slack.monicahq.com/badge.svg"></a>
+</p>
```
Comment (asbiin) [inline README.md:11]:
```suggestion
<p align="center">

![Monica's Logo](https://user-images.githubusercontent.com/61099/37693034-5783b3d6-2c93-11e8-80ea-bd78438dcd51.png)

<p>
<h1 align="center">Personal Relationship Manager</h1>

<div align="center">

[![Build Status](https://img.shields.io/azure-devops/build/monicahq/monica/monica%20CI/master)](https://dev.azure.com/monicahq/monica)
[![Docker pulls](https://img.shields.io/docker/pulls/library/monica)](https://hub.docker.com/_/monica/)
![Lines of code](https://img.shields.io/tokei/lines/github/monicahq/monica)
[![Code coverage](https://img.shields.io/sonar/coverage/monica?server=https%3A%2F%2Fsonarcloud.io)](https://sonarcloud.io/project/activity?custom_metrics=coverage&amp;graph=custom&amp;id=monica)
[![License](https://img.shields.io/github/license/monicahq/monica)](https://github.com/monicahq/monica/blob/master/LICENSE)

</div>
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,304 @@
...
+
+Here what we should do in order to realize our vision:
+
+- (**done**) Build an API in order to create an ecosystem. The ecosystem is what will make Monica a successful platform.
+
+- (**done**) Build importers and exporters of data. We don’t want to have any vendor lock-ins. Data is the property of the users and they should be able to do whatever they want with it.
+
+- Be the central point of contact management, by supporting CardDav protocol.
+
+- Be the central point of calendar events, by supporting CalDav protocol.
```
Comment (asbiin) [inline README.md:219]:
```suggestion
- (**done**) Be the central point of contact management, by supporting CardDav protocol.

- (**done**) Be the central point of calendar events, by supporting CalDav protocol.
```

**Comment by asbiin** [comment]
Thank you again @mgrhm

**Pull request merged**